stanford-sw/sw_index.properties

```diff
@@ -177,6 +177,8 @@ building_facet = custom, getBuildings, library_map.properties
 item_display = customDeleteRecordIfFieldEmpty, getItemDisplay
 
 mhld_display = custom, getMhldDisplay
+bookplates_display = custom, getBookplatesDisplay
+fund_facet = custom, getFundFacet

```

Comment:
D'oh! Nevermind.  I see that GitHub just doesn't include the changes to `StanfordIndexer.java` above.  I just scanned the document for the functions being defined. My bad.